app/models/host.rb

```diff
@@ -323,10 +323,6 @@ def validate_check_compliance_queue
     {:available => true, :message => nil}
   end
 
-  def validate_reset
-    validate_unsupported("Reset is unavailable")
-  end
```

Comment:
this was put in there because they replaced validate_reset with supports :reset, but some places still called `is_available?` (read: the whole reason why I created this PR)